src/layout/Tabs/config.ts

```diff
@@ -0,0 +1,61 @@
+import { CG, Variant } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Container,
+  rendersWithLabel: false,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInTabs: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+  },
+})
+  .addTextResource(
+    new CG.trb({
+      name: 'ariaLabel',
+      title: 'Aria Label',
+      description: 'The aria label for the tabs',
+    }),
+  )
+  .addProperty(new CG.prop('size', new CG.enum('small', 'medium', 'large').optional()))
+  .addProperty(new CG.prop('defaultTab', new CG.str().optional()))
+  .addProperty(
+    new CG.prop(
+      'tabs',
+      new CG.arr(
+        new CG.obj(
+          new CG.prop('id', new CG.str()),
```

Comment:
Did you consider making `tabs` a key/value map instead, where the key is the tab ID? Or possibly making it an array without an `id` and using indexes instead? It would of course be nice to have some validation or error-handling if you happen to configure two tabs with the same ID, which would not be possible to do with any of the other solutions.

Reply:
I have considered using indexes instead, but then you would not get to add validation or error handling. Could/would we add generated Ids for tabs to avoid similar id issues?

Reply:
Hmm, not sure what you mean - in what way does validation or error handling not work when using indexes? I'm not aware of any such limitations (i.e. `Grid` is using arrays instead of objects with IDs).

> Could/would we add generated Ids[...]?

Studio can, and should, generate unique IDs for tabs, yes. However, app developers working on plain layout files cannot. We could make the `id` property optional and generate one if it's missing - but then I would argue that it's entirely useless functionality (when the index is already exactly that - a guaranteed unique identifier for each tabs in the array of tabs).